docs/docs/admin/teamforcapella/full-setup/full-setup.md

```diff
@@ -0,0 +1,28 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB InfraGO AG and contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+This tutorial will guide you through the full setup of a new TeamForCapella
+based project in the Collaboration Manager.
+
+1. Create a new project:
+   [Create a new project](../../../user/projects/create.md)
+1. [Create new tool model](../../../user/projects/models/create.md), select
+   Capella as tool and the version you want to use.
+1. Make sure that right version and nature is selected! If they are not
+   configured, the following steps will not work!
+1. [Link the TeamForCapella repository to the new model](../project-integration/project-integration.md)
+   if not already done.
```

Comment:
I'd swap out the "if not alread done"s for "if you have not done so already", because it emphasizes that these are actions the user themself needs to take